sdk/storage/Azure.Storage.Blobs/src/BlobErrors.cs

```diff
@@ -31,6 +31,14 @@ internal static void VerifyHttpsCustomerProvidedKey(Uri uri, CustomerProvidedKey
             }
         }
 
+        internal static void VerifyCpkAndEncryptionScopeNotBothSet(CustomerProvidedKey? customerProvidedKey, string encryptionScope)
+        {
+            if (customerProvidedKey.HasValue && encryptionScope != null)
+            {
+                throw new ArgumentException("Customer provided key and encryption scope cannot both be set");
```

Comment:
Should we write this as `CustomerProvidedKey and EncryptionScope cannot both be set` to map a little closer to what people set on `BlobClientOptions`?

Reply:
(Can defer)

Reply:
fixed.